.github/workflows/scala.yml

```diff
@@ -27,6 +27,12 @@ jobs:
           sbt -v clean packageBin +test
           # sbt "project member" -v clean packageBin +test
 
+      - name: Build & Run Integration Tests
+        run: |
+          docker pull scylladb/scylla
+          sbt clean compile && sbt docker:stage && sbt docker:publishLocal
+          sbt -v clean packageBin IntegrationTest/test
+
       - name: Cleanup Before Caching
```

Comment:
I think you should refactor this. The code doesn't need to be compiled three times in two tasks. 
you should create a "build" task that does sbt -v clean Test/compile IntegrationTest/compile
Then create a task to run the basic tests, and integrationTests separately.